tests/test_check_for_empty_zones.py

```diff
@@ -0,0 +1,96 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+from check_empty_zones import get_aws_zones, is_zone_empty, main
+
+
+@pytest.fixture
+def mock_boto3_client():
+    with patch('boto3.client') as mock_client:
+        yield mock_client
```

Comment:
This feels like a good method to use for testing at scale 🚀 

I believe sticking with the explicit patches on each test is more readable in this case 👀 But maybe I'm stuck in my ways 😅